test/models/FileRoleSpec.scala

```diff
@@ -39,7 +39,7 @@ class FileRoleSpec extends SpecBase {
     }
 
     "return a JsSuccess for CashStatement" in {
```

Comment:
Could you please replace CashStatement with CDSCashAccount?

Reply:
Replaced CashStatement with CDSCashAccount